README.md

```diff
@@ -41,37 +45,61 @@
 		
 </table>
 
-### How to Run Locally
-Run the below command to clone the repo locally
+## How to Contribute:
+- <b>Fork the Project Repository :</b><br>
+You can make a copy of the project to your account in order to make contribution.This process is called forking a project to your Github account.Click on the fork button to fork the project.
+
+![Screenshot (247)](https://user-images.githubusercontent.com/61612939/113122940-bd2fa080-9231-11eb-9c62-61d2eb711f7c.png)
+
+- <b>Clone the Project Repository :</b><br>
+To get this project on your development machine we use clone command of git.Run the below command to clone the repo locally
 ```
 git clone https://github.com/kothariji/SyntaxMeets.git
 ``` 
 
-### To run the server, go to the server folder in the backend directory and Run 
+- <b>Add a Remote the Original Project Repository: </b><br>
+Remote means the remote location of project on Github.This will point to the original repository. It will help you to get the latest changes made by others.
+```
+$ cd <your-forked-project-folder>
+$ git remote add upstream https://github.com/<author-account-username>/<project>.git
+``` 
```

Comment:
Don't you think, giving details about how **remote** works in the readme is good practice?

Reply:
I think, giving details about how remote works in the readme is good practice because if we consider a beginner who wants to contribute to the project but have very less or no knowledge of git ,it would become easier for him/her to make his contributions effectively if he/she gets all the info in the project's readme itself and does not use/require any other source for making contributions to the project. So, it would be effective if we provide a short info about remote in the readme itself, taking a beginner under consideration.

Reply:
Yes, I agree with you, So these are contributing guidelines right. So IMO it should come under Contributing.md and not readme.md, What Say?

Reply:
Yes, you are right, these should come under Contributing. md. But, I think that adding these guidelines in the readme. MD would help the contributors get a quick glimpse of how they can contribute by just reading the readme. md file only. But still, if you suggest I'll make amendments accordingly to Contributing. md file.